systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/AbstractKafkaClient.java

```diff
@@ -108,7 +108,7 @@ protected AbstractKafkaClient(Builder<?> builder) {
         if (builder.topicName == null || builder.topicName.isEmpty()) throw new InvalidParameterException("Topic name is not set.");
         if (builder.namespaceName == null || builder.namespaceName.isEmpty()) throw new InvalidParameterException("Namespace name is not set.");
         if (builder.clusterName == null  || builder.clusterName.isEmpty()) throw  new InvalidParameterException("Cluster name is not set.");
-        if (builder.messageCount <= 0) throw  new InvalidParameterException("Message count is less than 1");
+        if (builder.messageCount == 0) throw  new InvalidParameterException("Message count is less than 1");
```

Comment:
is that change desired? It open doors for troubles I think.

Reply:
Well @see-quick had comment that we never want to send less than one message. So I changed it in both example clients and `AbstractKafkaClient`.. I think that better it will be with the `<=` :)

Reply:
I did not mean to change it to `==`. I thought that there was `builder.messageCount < 0)` condition which allowes you to send `0 messages`. That's what I was referring to. Anyway `builder.messageCount <= 0)` is the correct one

Reply:
I thought that there was `(builder.messageCount < 0)` which means you can also send `0` messages. That's was I was referring to. Anyway, `(builder.messageCount <= 0)` is the best option here.